src/genie/libs/parser/iosxe/tests/ShowWlanAll/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,340 @@
+    "wlan_names": {
+        "lizzard_Fabric_F_cf6efda4": {
+            "identifier": 17,
+            "ssid": "lizzard",
+            "status": "Enabled",
+            "broadcast_ssid": "Enabled",
+            "advertise_ap_name": "Disabled",
+            "universal_ap_admin": "Disabled",
+            "max_clients_wlan": 0,
+            "max_clients_ap": 0,
+            "max_clients_ap_radio": 200,
+            "okc": "Enabled",
+            "active_clients": 4,
+            "chd_per_wlan": "Enabled",
+            "wmm": "Allowed",
+            "channel_scan_defer_priority": {
+                "priority": [
+                    5,
+                    6
+                ]
+            },
+            "scan_defer_time_msecs": 100,
+            "media_stream_multicast_direct": "Disabled",
+            "ccx_aironet_support": "Disabled",
+            "p2p_blocking_action": "Disabled",
+            "radio_policy": "802.11a only",
+            "local_eap_authentication": "Disabled",
+            "mac_filter_authorization_list_name": "Disabled",
+            "mac_filter_override_authorization_list_name": "Disabled",
+            "dot1x_authentication_list_name": "dnac-list",
+            "dot1x_authorization_list_name": "Disabled",
+            "security": {
+                "dot11_authentication": "Open System",
+                "static_wep_keys": "Disabled",
+                "wifi_protected_access_wpa1_wpa2_wpa3": "Enabled",
+                "wifi_protected_access_wpa": {
+                    "wpa_ssn_ie": "Disabled"
+                },
+                "wifi_protected_access_wpa2": {
+                    "wpa2_rsn_ie": "Enabled",
+                    "wpa2_attributes": {
+                        "mpsk": "Disabled",
+                        "aes": "Enabled",
+                        "ccmp256": "Disabled",
+                        "gcmp128": "Disabled",
+                        "gcmp256": "Disabled",
+                        "randomized_gtk": "Disabled"
+                    }
+                },
+                "wifi_protected_access_wpa3": {
+                    "wpa3_ie": "Disabled"
+                },
+                "auth_key_mgmt": {
+                    "dot1x": "Enabled",
+                    "psk": "Disabled",
+                    "cckm": "Disabled",
+                    "ft_dot1x": "Enabled",
+                    "ft_psk": "Disabled",
+                    "dot1x_sha256": "Disabled",
+                    "psk_sha256": "Disabled",
+                    "sae": "Disabled",
+                    "owe": "Disabled",
+                    "suiteb_1x": "Disabled",
+                    "suiteb192_1x": "Disabled"
+                },
+                "owe_transition_mode": "Disabled",
+                "osen": "Disabled",
+                "ft_support": "Enabled",
+                "ft_support_attributes": {
+                    "ft_reassociation_timer_secs": 20,
+                    "ft_over_the_ds_mode": "Disabled"
+                },
```

Comment:
for this kind of case, I understand your approach, but better to have all under same key. so that it will be easy to get by key `ft_support`. if you agree, please change similar places as well.
```suggestion
                "ft_support": {
                    "ft_support_status": "Enabled",
                    "ft_reassociation_timer_secs": 20,
                    "ft_over_the_ds_mode": "Disabled"
                },
```

Reply:
I like your idea. I updated this one as well as everything below it. Please double check. Thanks!